uint64 track_id
string track_label
uint32 min_embeddings
time expiration
bool reset_label
---
bool success
